aya/src/sys/bpf.rs

```diff
@@ -468,17 +468,21 @@ pub(crate) fn bpf_prog_get_fd_by_id(prog_id: u32) -> Result<OwnedFd, io::Error>
     unsafe { fd_sys_bpf(bpf_cmd::BPF_PROG_GET_FD_BY_ID, &attr).map_err(|(_, e)| e) }
 }
 
-pub(crate) fn bpf_prog_get_info_by_fd(prog_fd: RawFd) -> Result<bpf_prog_info, io::Error> {
+pub(crate) fn bpf_prog_get_info_by_fd(
+    prog_fd: RawFd,
+    map_ids: &[u32],
+) -> Result<bpf_prog_info, io::Error> {
     let mut attr = unsafe { mem::zeroed::<bpf_attr>() };
-    // info gets entirely populated by the kernel
-    let info = MaybeUninit::zeroed();
+    let mut info = unsafe { mem::zeroed::<bpf_prog_info>() };
```

Comment:
why switch from MaybeUninit to mem::zeroed()?

Reply:
FYI this landed in https://github.com/aya-rs/aya/pull/747. We could use either `mem::zeroed()` or `MaybeUninit::zeroed().assumit_init()` since we have to write into the thing. We chose the former, but they're equivalent per the `mem::zeroed` [documentation](https://doc.rust-lang.org/stable/std/mem/fn.zeroed.html).